test/plugin/scenarios/webflux-scenario/webflux-webclient-scenario/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+server.port: 18081
```

Comment:
Should the format be `k=v` in properties?

Reply:
> Should the format be `k=v` in properties?

Thank you very muh for pointing my mistake.Originally, I wanted to write configuration file imitating `webflux-webclient-scenario`, but I ignore the difference in file type . I will fix it immediately.